src/app/component/AddToCart/AddToCart.container.js

```diff
@@ -132,6 +133,11 @@ export class AddToCartContainer extends PureComponent {
         }).then(() => this._afterAdded());
     }
 
+    disabledButtonClick(event){
+        const { showNotification } = this.props;
+        showNotification('success', 'Please, choose desired item properties');
```

Comment:
This is not a `success` case. More like `error`.

Reply:
The string you provided is not translatable. Make sure it is by wrapping in `__(<MY_STRING>)`.

Reply:
The string should state: `Product options are not configured!`.

Reply:
Please separate this PR into multiple. This is needed as it seems like some changes might not get to production - we need product owner approval.